http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpConfigurationOption.java

```diff
@@ -59,6 +59,12 @@
     public static final SdkHttpConfigurationOption<Duration> CONNECTION_MAX_IDLE_TIMEOUT =
             new SdkHttpConfigurationOption<>("ConnectionMaxIdleTimeout", Duration.class);
 
+    /**
+     * Whether or not to use keepalive on the connection.
+     */
+    public static final SdkHttpConfigurationOption<Boolean> CONNECTION_KEEPALIVE =
```

Comment:
Can we rename it to `TCP_KEEPALIVE` so that users don't confuse it with HTTP keep alive?